protocol/implementations/js/src/protocolDefinitions.ts

```diff
@@ -0,0 +1,121 @@
+                'application/json'
+            ]
+        },
+        'Order': {
+            'schema': 'https://tbd.website/protocols/tbdex/Order',
+            'dataFormats': [
+                'application/json'
+            ]
+        },
+        'OrderStatus': {
+            'schema': 'https://tbd.website/protocols/tbdex/OrderStatus',
+            'dataFormats': [
+                'application/json'
+            ]
+        }
+    },
+    'structure': {
+        // alice sends RFQs, not receives them
+        'RFQ': {
+            '$actions': [],
+            // whoever received the RFQ that Alice sent, can write back a Quote to Alice
+            'Quote': {
+                '$actions': [
+                    {
+                        'who': 'recipient',
+                        'of': 'RFQ',
+                        'can': 'write'
+                    }
+                ],
+                // alice sends Orders, not receives them
+                'Order': {
+                    '$actions': [],
+                    // whoever received the order that Alice sent in response to a Quote in response to an RFQ, can write back an OrderStatus to Alice.
+                    'OrderStatus': {
+                        '$actions': [
+                            {
+                                'who': 'recipient',
+                                'of': 'RFQ/Quote/Order',
+                                'can': 'write'
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    }
+};
+
+
+export const pfiProtocolDefinition = {
+    'protocol': 'https://tbd.website/protocols/tbdex',
+    'types': {
+        'RFQ': {
+            'schema': 'https://tbd.website/protocols/tbdex/RequestForQuote',
+            'dataFormats': [
+                'application/json'
+            ]
+        },
+        'Quote': {
+            'schema': 'https://tbd.website/protocols/tbdex/Quote',
+            'dataFormats': [
+                'application/json'
+            ]
+        },
+        'Order': {
+            'schema': 'https://tbd.website/protocols/tbdex/Order',
+            'dataFormats': [
+                'application/json'
+            ]
+        },
+        'OrderStatus': {
+            'schema': 'https://tbd.website/protocols/tbdex/OrderStatus',
+            'dataFormats': [
+                'application/json'
+            ]
+        }
+    },
+    'structure': {
+        // anyone can write RFQ to a PFI's DWN
+        // no one can read RFQs from PFI's DWN (except the PFI itself)
+        'RFQ': {
+            '$actions': [
+                {
+                    'who': 'anyone',
+                    'can': 'write'
+                }
+            ],
+            // PFI is sending OUT quotes. no one should be writing Quotes to PFIs.
+            'Quote': {
+                '$actions': [],
+                // PFI who received an RFQ/Quote (i.e. Alice) can write Order to PFI's DWN
```

Comment:
>PFI who received an RFQ/Quote (i.e. Alice)

Alice is a PFI?

Reply:
I think that is a typo @jiyoontbd should be "Customer who received quote" as don't customers get quotes in response to RFQs? PFIs get RFQs and emit quotes in response.

Reply:
> Alice is a PFI?

nope, i just can't write good like. apologies for the confusion!

what i meant by `PFI who received an RFQ/Quote (i.e. Alice)` was actually:

`only Alice, who received an RFQ/Quote, can write Order to PFI's DWN`

also, as an addendum, here i wrote the word `RFQ/Quote` to refer to `Quote` that Alice received from the PFI who received Alice's RFQ, in the `protocolPath` lingo, like how i refer to it in the `of: 'RFQ/Quote'`.